i18n/nl/cosmic_settings.ftl

```diff
@@ -0,0 +1,807 @@
+    .desc = Bluetooth-apparaten bewerken
+    .status = Dit systeem is zichtbaar als { $aliases } zolang de Bluetooth-instellingen open staan.
+    .connected = Verbonden
+    .connecting = Verbinding maken…
+    .disconnecting = Verbinding verbreken…
+    .connect = Verbinden
+    .disconnect = Verbinding verbreken
+    .forget = Vergeten
+    .dbus-error = Er is een fout opgetreden tijdens de interactie met DBus: { $why }
+
+bluetooth-paired = Eerder verbonden apparaten
+    .connect = Verbinden
+    .battery = { $percentage }% batterij
+
+bluetooth-confirm-pin = Bluetooth-pincode bevestigen
+    .description = Controleer of de volgende pincode overeenkomt met de pincode die op { $device } wordt weergegeven
+
+bluetooth-available = Nabije bluetooth-apparaten
+
+bluetooth-adapters = Bluetooth-adapters
+
+## Desktop
+
+desktop = Bureaublad
+
+## Desktop: Wallpaper
+
+wallpaper = Schermachtergrond
+    .change = Wijzig alle afbeeldingen
+        <#-- Wijzig alle afbeeldingen: Change every image -->
+    .desc = Opties voor schermachtergrond, kleuren en diavoorstellingen
+    .fit = Schermachtergrond aanpassen
+    .folder-dialog = Selecteer map met achtergronden
+    .image-dialog = Selecteer schermachtergrond
+    .plural = Achtergronden
+    .same = Dezelfde achtergrond op alle schermen
+    .slide = Diavoorstelling
+
+add-color = Kleur toevoegen
+add-image = Afbeelding toevoegen
+all-displays = Alle beeldschermen
+colors = Kleuren
+dialog-add = Toevoegen
+fill = Vullen
+fit-to-screen = Aan het scherm passend maken
+open-new-folder = Nieuwe map openen
+recent-folders = Recente mappen
+
+x-minutes = { $number } minuten
+x-hours = { $number } uur
+<#-- the plural of hour in Dutch, doesn't get used when a number is in front of it. We say: '4 uur lang', not '4 uren lang' -->
+
+never = Nooit
+
+## Desktop: Appearance
+
+appearance = Uiterlijk
+    .desc = Kleuren en thema's.
+
+accent-color = Accentkleur
+app-background = Achtergronden van applicaties of vensters
+auto = Automatisch
+close = Sluiten
+color-picker = Kleurkiezer
+copied-to-clipboard = Naar klembord gekopieerd
+copy-to-clipboard = Naar klembord kopiëren
+dark = Donker
+export = Exporteren
+hex = Hex
+import = Importeren
+light = Licht
+mode-and-colors = Kleuren en lichte/donkere modus
+recent-colors = Recente kleuren
+reset-to-default = Naar standaardinstellingen terugzetten
+rgb = RGB
+window-hint-accent = Accentkleur als visuele hint voor het actieve venster
+window-hint-accent-toggle = Gebruik de accentkleur van het thema als visuele hint voor het actieve venster
+
+auto-switch = Automatisch wisselen tussen lichte en donkere modus
+    .sunrise = Wissel naar lichte modus bij zonsopgang
+    .sunset = Wissel naar donkere modus bij zonsondergang
+    .next-sunrise = Wissel naar lichte modus bij de volgende zonsopgang
+    .next-sunset = Wissel naar donkere modus bij de volgende zonsondergang
+
+<#-- not sure about these translations, don't know what is meant with Container color -->
```

Comment:
line 193 to 195: I don't know what container colors are, so I can't translate, In my opinion